### PR TITLE
RLPT-123 change data

### PR DESCRIPTION
**Pull request by roxhens**

### CODE REVIEW

File: src/data/data.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "About": {
     "paragraph": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
     "Why": [
-      "Lorem ipsum dolor",
+      "I am changing this text",
```
Comment (roxhens) [inline src/data/data.json:9]: Why did you change this? 
also this needs a dot at the end

File: src/data/data.json
Comment (Luksuz) [inline src/data/data.json:1]: Why did you add this?
Also add the dot to the end.

File: src/data/data.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "About": {
     "paragraph": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
     "Why": [
-      "Lorem ipsum dolor",
```
Comment (Luksuz) [inline src/data/data.json:9]: Why did you add this?
Also add the dot to the end.

File: src/data/data.json
Comment (roxhens) [reply src/data/data.json:1]: hey this is done

File: src/data/data.json (hunk 1 above)
Comment (Luksuz) [inline src/data/data.json:9]: Why did you add this?
Also add the dot to the end.

**Review by Luksuz — CHANGES_REQUESTED** [review]
Fix this

**Comment by roxhens** [comment]
> Fix this

fixed

**Review by Luksuz — APPROVED** [review]
